src/components/Error.jsx

```diff
@@ -118,9 +126,14 @@ export const BZReportModal = ({
                 ));
     }, []);
 
+    const {
+        PRETTY_NAME: prettyName,
+    } = useContext(OsReleaseContext);
+
     const openBZIssue = (reportURL) => {
         reportURL = ensureMaximumReportURLLength(reportURL);
         reportURL = addLogAttachmentCommentToReportURL(reportURL);
+        reportURL = addOSPrettyNameCommentToReportURL(reportURL, prettyName);
```

Comment:
It appears to the top now, I want the log attachment message to be appeating on the top, as it requires user intervention and it should stand out.

Reply:
ok! I modified to appear on the bottom